src/app/Instance/CreateInstance/CreateInstance.test.tsx

```diff
@@ -347,6 +379,20 @@ const setName = (comp: RenderResult, name: string): void => {
   });
 };
 
+const setErrorHandlginMethod = async (
```

Comment:
const setErrorHandlginMethod = async (
```suggestion
const setErrorHandlingMethod = async (
```